WcaOnRails/app/models/competition.rb

```diff
@@ -699,15 +699,15 @@ def psych_sheet_event(event)
 
     prev_registration = nil
     registrations.each_with_index do |registration, i|
-      if event.recommended_format.sort_by == :single
+      if sort_by == 'single'
```

Comment:
Changing this from a symbol to a string doesn't seem like it would be without side effects... how does this still work?